app/src/app/names/page.tsx

```diff
@@ -77,7 +77,10 @@ export default async function NameChangesPage(props: PageProps) {
                     <ArrowRightIcon className="h-4 w-4 text-white" />
                   </ListTableCell>
                   <ListTableCell className="text-sm font-medium text-white">
-                    {nameChange.newName}
+                    {nameChange.status === "approved" ? (
```

Comment:
Let's use a <Link> component instead (it prevents a full page refresh)

Note: I added some hover styling to the link element

```jsx
{nameChange.status === "approved" ? (
  <Link
    href={`/players/${nameChange.newName}`}
    className="hover:text-gray-100 hover:underline"
  >
    {nameChange.newName}
  </Link>
) : (
  <span>{nameChange.newName}</span>
)}
```